src/modules/views/Sharings/index.jsx

```diff
@@ -84,6 +88,22 @@ export const SharingsView = ({ sharedDocumentIds = [] }) => {
   )
   const result = useQuery(query.definition, query.options)
 
+  const filteredResult = useMemo(
+    () => ({
+      ...result,
```

Comment:
Why do you need to keep `results` in this object?

Reply:
In `FolderViewBody` component, it uses some fields in result so I think it's better to keep it